### PR TITLE
UI #8513: UI Improvements - 0.13 - part 2

### PR DESCRIPTION
**Pull request by aniketkatkar97**
### Describe your changes :

I worked on 
- Converted normal table to antd table for 'Service Page'
- Replaced old PageContainer with new one for 'Data Asset Details' , 'Service', 'Database Schema Details', 'Database Details' pages
- Table styling made consistent throughout the application
- Fixed the delete icon issue on the UsersList page
- Code optimizations

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">

Fixed delete button cutting up issue:
Screenshot 2022-11-16 at 6 54 11 PM

Table Details Page: 
Screenshot 2022-11-16 at 6 54 45 PM

Topic Details Page: 
Screenshot 2022-11-16 at 6 54 56 PM

Dashboard Details Page:
Screenshot 2022-11-16 at 6 55 09 PM

Pipeline Details Page: 
Screenshot 2022-11-16 at 6 55 19 PM

Service Details Page: 
Screenshot 2022-11-16 at 7 02 41 PM

Database Details Page:
Screenshot 2022-11-16 at 7 02 48 PM

Database Schema Details Page:
Screenshot 2022-11-16 at 7 02 53 PM
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Frontend: @open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx [hunk 1]
```diff
@@ -846,7 +846,9 @@ const PipelineDetails = ({
                 {t('label.tasks')}
               </span>
             }>
-            <Row gutter={[16, 16]}>
+            <Row
+              className="tw-bg-white tw-flex-grow tw-p-4 tw-shadow tw-rounded-md"
+              gutter={[0, 16]}>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx:851]: Should use Card

File: openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/DatabaseSchemaPage.component.tsx [hunk 2]
```diff
@@ -640,29 +642,31 @@ const DatabaseSchemaPage: FunctionComponent = () => {
...
-                    entityType={EntityType.DATABASE_SCHEMA}
-                  />
-                </Space>
-
-                <div className="tw-flex tw-gap-1 tw-mb-2 tw-mt-1 tw-ml-7 tw-flex-wrap">
+            <PageContainerV1>
+              <Row
+                className="p-x-md p-t-lg"
+                data-testid="page-container"
+                gutter={[0, 12]}>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/DatabaseSchemaPage.component.tsx:649]: Card can be used here?

**Comment by aniketkatkar97** [comment]
@chirag-madlani I worked on the comments and pushed the changes. Can you please review it?

**Review by ShaileshParmar11 — DISMISSED** [review]

**Review by ShaileshParmar11 — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/styles/components/table.less [hunk 3]
```diff
@@ -9,14 +9,21 @@
...
 }
 
+.table-shadow {
+  .ant-table-bordered {
+    table {
+      box-shadow: 1px 1px 3px 0px rgba(0, 0, 0, 0.12);
+    }
+  }
+}
+
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/styles/components/table.less:26]: Table has shadow already ASAIK

File: openmetadata-ui/src/main/resources/ui/src/pages/service/index.tsx [hunk 4]
```diff
@@ -1052,171 +1035,129 @@ const ServicePage: FunctionComponent = () => {
...
+                                (serviceCategory ===
+                                  ServiceCategory.METADATA_SERVICES &&
+                                  serviceFQN === OPENMETADATA)
+                              }
+                              loading={isTestingConnection}
                               type="primary"
-                              onClick={handleEditConnection}>
-                              Edit Connection
+                              onClick={checkTestConnect}>
+                              Test Connection
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/service/index.tsx:1146]: localisation

File: openmetadata-ui/src/main/resources/ui/src/pages/service/index.tsx [hunk 5]
```diff
@@ -1052,171 +1035,129 @@ const ServicePage: FunctionComponent = () => {
...
+                            onClick={handleEditConnection}>
+                            Edit Connection
+                          </Button>
+                        </Tooltip>
+                        {allowTestConn && isAirflowRunning && (
                           <Tooltip
                             title={
                               servicePermission.EditAll
-                                ? 'Edit Connection'
+                                ? 'Test Connection'
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/service/index.tsx:1131]: localisation

File: openmetadata-ui/src/main/resources/ui/src/pages/service/index.tsx [hunk 6]
```diff
@@ -1052,171 +1035,129 @@ const ServicePage: FunctionComponent = () => {
...
+
+                  {activeTab === 2 && getIngestionTab()}
+
+                  {activeTab === 3 && (
+                    <>
+                      <Space className="w-full my-4 justify-end">
+                        <Tooltip
+                          title={
+                            servicePermission.EditAll
+                              ? 'Edit Connection'
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/service/index.tsx:1115]: localisation

File: openmetadata-ui/src/main/resources/ui/src/pages/service/index.tsx [hunk 7]
```diff
@@ -971,8 +886,77 @@ const ServicePage: FunctionComponent = () => {
...
+            <Link to={getLinkForFqn(record.fullyQualifiedName || '')}>
+              {isUndefined(text)
+                ? getEntityName(record as unknown as EntityReference)
+                : text}
+            </Link>
+          );
+        },
+      },
+      {
+        title: 'Description',
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/service/index.tsx:930]: localisation

File: openmetadata-ui/src/main/resources/ui/src/styles/components/table.less (hunk 3 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/styles/components/table.less:26]: Table shadows were applied globally, I moved it to a global class 'table-shadow' to give shadows only to the tables we want

**Comment by aniketkatkar97** [comment]
@chirag-madlani made localization changes for the files. Can you please review?

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**